docker-compose.yml

```diff
@@ -5,45 +5,46 @@ services:
     build:
       context: .
       dockerfile: ./build/Dockerfile
+    restart: always
     ports:
       - "8080:8080"
     depends_on:
       - db
-    links:
-      - db
     networks:
-      - asgard
+      - asgard_mail
+      - asgard_db
     environment:
       APP_DOMAIN: ".127.0.0.1"
       COOKIE_SECURE: "false"
       SMTP_SECURE: "false"
       SMTP_HOST: mail
+      DB_NAME: thunderdome
+      DB_USER: thor
+      DB_PASS: odinson
     volumes:
-      - ./etc:/etc/thunderdome
+      - ./data/thunderdome:/etc/thunderdome
```

Comment:
This isn't a named volume because its meant to carry local config into container